repl

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
```

Comment:
This script should go to `scripts`

Reply:
Oh, `scripts/` doesn't exist yet. Are you sure? Alternatively it can go in the `cachix/` directory, which it is used for.

Reply:
Been a while, seems ok to keep it top-level for now.